### ISSUE TITLE
[Vertical Tabs] Add the ability to double click the empty space to open a new tab

### ISSUE THREAD

**Issue by TheNightRider12** [brave/brave-browser#32097]
## Description 
The only current way to open a new tab with Vertical Tabs enabled is to click on "+ New Tab". I think having the option to double click the empty space to open a new tab would be neat. A quicker way to do so while still having the option to use the New Tab button as well. 

## Steps to Reproduce
N/A

## Actual result:
N/A

## Expected result:
N/A

## Other Additional Information:
There is an extension on Firefox to enable a Vertical Tabs like feature for FF called Sideberry that has either the option enabled by default or is an option within the extension. Which allows double clicking in the empty space to open a new tab.

**Comment by StrangePeanut** [comment]
I think it would make much more sense to reposition the new tab button to be directly below the tabs.

**Comment by TheNightRider12** [comment]
> I think it would make much more sense to reposition the new tab button to be directly below the tabs.

I like that idea too. Something to make opening a new tab quicker.

**Comment by sangwoo108** [comment]
cc. @rebron , @aguscruiz I'd like to hear from you. I like the idea and also I don't thin we  have risk to try this.

**Comment by sangwoo108** [comment]
> 
I think it would make much more sense to reposition the new tab button to be directly below the tabs.


This was our intentional decision so that users can open multiple tabs without moving their mouse. But we're always open to revisit that 😄

**Comment by StrangePeanut** [comment]
@sangwoo108 thanks for your reply. When would one ever need to open multiple new tabs at once? I might be missing something but I can't think of any scenario. If the change wouldn't involve a lot of refactoring, how about making it into a toggle option in the right-click context menu?

Just to clarify that I wasn't criticising TheNightRider12's suggestion and essentially hold the same opinion: Opening new vertical tabs by mouse could be made more intuitive.

**Comment by sangwoo108** [comment]
> When would one ever need to open multiple new tabs at once

Good point. I agree with you on that 😄 but still agree with what @aguscruiz says at the same time - in the point of muscle memory point, we can just throw away cursor to the corner of screen to press the button.

**Comment by TheNightRider12** [comment]
I can see how having the new tab button being below the tabs themselves could be an issue. Especially as you pointed out if you have scrollable tabs. I think having a buffer area below the last tab in the list to where the "space" detects a double click to then open a new tab might be a good workaround. Just to prevent misclicks.

**Comment by tjnycum** [comment]
There's an option that would continue to have the cited advantages of the new-tab button's current positioning, while also addressing this request: Make the height/width of the new-tab button elastic (with a minimum of its current height/width), so that it expands to take up any slack space in the tab list. That way people could click anywhere in the otherwise non-functional space to open a new tab, including just after the last tab or at the extreme (using muscle memory).

Appearance-wise, the simplest would be to just center the button's existing labeling along the axis that's newly variable in length. I suspect that would look okay, with the existing separator bar and highlighting of the button on hover making the extent of the button clear, but it might look better in the case of the expanded vertical tab bar to orient the "New tab" (and keyboard shortcut) text vertically and enlarge it to take up more of the space. If the text were too distracting enlarged, perhaps make it fainter.

An optional addition:
Whenever any tabs are being dragged (in any Brave window), you could make the button switch to looking more like a drop zone, hiding (or changing) its label and giving it a thick dashed line border. Dropping tab(s) on it would be the same as positioning them at the end of the list.
